compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/window/PopupLayout.skiko.kt

```diff
@@ -107,12 +104,15 @@ internal fun PopupLayout(
         }
         owner to composition
     }
-    owner.density = density
     DisposableEffect(Unit) {
         onDispose {
             scene.detach(owner)
             composition.dispose()
             owner.dispose()
         }
     }
+    LaunchedEffect(density, layoutDirection) {
```

Comment:
LaunchedEffect will dispatch the lambda only after we draw the first frame. Better to use SideEfect here

Reply:
Good to know.